DemoApp/DemoAppConfiguration.swift

```diff
@@ -19,24 +18,15 @@ enum DemoAppConfiguration {
         #endif
     }
 
-    // MARK: Internal configuration
-
-    static var isStreamInternalConfiguration: Bool {
-        ProcessInfo.processInfo.environment["STREAM_DEV"] != nil
-    }
-
     // This function is called from `DemoAppCoordinator` before the Chat UI is created
     static func setInternalConfiguration() {
-        StreamRuntimeCheck.assertionsEnabled = isStreamInternalConfiguration
```

Comment:
Maybe we could move everything in one place? At the moment, we set `assertionsEnabled` here, while we have all the other setup in the demo app config.

Reply:
Done ✅